src/ui/button/Button.scss

```diff
@@ -1,7 +1,19 @@
 @import 'styles/mixins';
 
-.button,
-.iconButton {
+button {
+  box-sizing: border-box;
+  border: none;
+  outline: none;
+  background-color: transparent;
+  position: relative;
+  padding: 9px 20px !important;
```

Comment:
Why you used important if this is module without any parent stylings

Reply:
was fixed